Markdown/Markdown.sublime-syntax

```diff
@@ -203,10 +203,12 @@ contexts:
         - include: table
         - match: ''
           pop: true
-    - match: '^([ ]{0,3})([*+-])(?=\s)'
+    - match: ^([ ]{0,3})([*+-])( (\[[ xX]\]))?(?=\s)
       captures:
         1: markup.list.unnumbered.markdown
         2: markup.list.unnumbered.bullet.markdown punctuation.definition.list_item.markdown
+        3: markup.list.unnumbered.markdown
+        4: keyword.other.markdown.github
```

Comment:
[in other places in this syntax def](https://github.com/sublimehq/Packages/blob/8f997996e381ae473aaae0e6b0d6a137705a12b5/Markdown/Markdown.sublime-syntax#L768), we are using `.markdown-gfm` as the suffix to scope features specific to GitHub Formatted Markdown